src/tiler.coffee

```diff
@@ -21,130 +21,152 @@
 ) @, ($) ->
 
   $.widget 'ui.tiler',
-    #
-    # WIDGET SETUP/METHODS
-    #
     widgetEventPrefix: 'tiler'
     options:
+      activeTile: 1
       isReversible: true
-      startingActiveTile: 1
-      startingPreviousTile: 2
-
-    _create: ->
-      @currentActiveTileIndex = @options.startingActiveTile
-      @currentPreviousTileIndex = @options.startingPreviousTile
 
     _init: ->
-      @element.addClass 'animation-disabled'
-
       # Collect all the tiles, except for those nested inside another tiler instance
       @$tiles = $('.tiler-tile', @element).not(@element.find('.tiler-viewport .tiler-tile'))
-      @$enterTile = @$tiles.eq @currentActiveTileIndex - 1
-      @$exitTile = @$tiles.eq @currentPreviousTileIndex - 1
+      @$currentActiveTile ||= @_getTile @options.activeTile
```

Comment:
i know this is a ruby-ism but i still don't believe it translates correctly. compiled JS is:

`this.$currentActiveTile || (this.$currentActiveTile = this._getTile(this.options.activeTile));`

which is not what you want, right?